src/lib/TranscriptEditor/TimedTextEditor/WrapperBlock.module.css

```diff
@@ -1,44 +1,96 @@
 @value color-labs-red, color-light-grey, color-mid-grey, color-dark-grey from '../colours.module.css';
 
-.WrapperBlock {
-  height: 100%;
-  width: 100%;
-  margin-bottom: 2em;
-  line-height: 1.2em;
-  overflow: hidden;
-}
+/* Desktop size */
+@media (min-width: 768px) {
 
-.markers {
-  width: 25%;
-  font-size: 0.8em;
-  line-height: 1.2em;
-  float: left;
-}
+  .markers {
+    width: 25%;
+    font-size: 0.8em;
+    line-height: 1.2em;
+    float: left;
+  }
+
+  .WrapperBlock {
+    height: 100%;
```

Comment:
I think again this isn't doing anything (same comment for a fair few of the css additions)

Reply:
ok cool